rsp_vision/save/tables_manager.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+
+import pandas as pd
+
+
+class AnalysisSuccessTable:
```

Comment:
could add docstring here, I am not 100% sure exactly what this is doing before reading the code in more detail. Is this collating information in a `DataFrame`? Maybe `AnalysisSucessDataFrame` makes more sense as for some reason I am thinking it is a table formatter for the dashboard.